packages/sdk/identity/src/odis/identifier-backwards-compatibility.test.ts

```diff
@@ -0,0 +1,85 @@
+import { getPhoneHash } from '@celo/base'
```

Comment:
this test should be ignored in CI since they'll be prone to flakiness. We do this in the odis packages by using the '--testPathIgnorePatterns test/end-to-end' flag